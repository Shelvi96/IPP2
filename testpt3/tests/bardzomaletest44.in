NEW ID

458849030>57:7664
83;171;:5>02:
;;8380:02>85
637:;220>3;00342605
0150>66
6;:025>5;491;0
;455:9;3>001367:4;8
1>7;
38:7>;:18
3>7::;:369
5>1567
56086:33>281:3:91
0>765
@639424904:
@2527;281:::32271902
@;9533953172
@08642250
@8462;;;4283
@1:7;7:420070:;318398;4
@37279:21008;8:7:3;93:8:;
@281144
@07315565:28:54087:35:
@43859115:50176;13184;
@0;179410693664882612708:24
@28:7828734833:;947549;60
@:01481
@4
@5491
@160892;:774;4798:8104953608::
@23873
@9378:306397198125:9;;5213843
@705:050757:360425:;4
