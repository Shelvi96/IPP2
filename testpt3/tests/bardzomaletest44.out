0
211000
0
0
0
111000100
3917258172
0
22100100
60555429
6588109406417
82528344998
0
0
0
49003022289567744
0
4083585190797312
1180989
